recipe/meta.yaml

```
@@ -25,6 +25,9 @@ requirements:
     - pandas
     - matplotlib-base
     - scipy
+    - plotly
+    - statsmodels
+    - rdkit
```

```suggestion
    - rdkit
    - versioneer
```
I don't _think_ you need to have versioneer in your depenencies if you vendored, but erm.. give this a try?